zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java

```diff
@@ -43,6 +44,7 @@
  */
 @RestController
 @RequestMapping("/api/v1")
+@CrossOrigin("${zipkin.server.allowed-origins:''}")
```

Comment:
Making the default value here also `*` would be a good idea for those making custom Zipkin servers with `@EnableZipkinServer` or those using their own YAML file of properties that might not contain the new property.
That is, if we want the default value to be `*` in these cases also. Do we? Or do we want people in those situations to make a conscious decision about the allowed origins for the query API?